src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/ToolEnvironment.java

```diff
@@ -144,7 +142,6 @@ protected ToolEnvironment(Context context) {
         finder = JavadocClassFinder.instance(context);
         enter = JavadocEnter.instance(context);
         names = Names.instance(context);
-        externalizableSym = syms.enterClass(syms.java_base, names.fromString("java.io.Externalizable"));
```

Comment:
What was externalizableSym about and why have you deleted it?

Reply:
TL;DR: the value was unused.  It's gratuitous cleanup ;-)

The value was the `Symbol` (javac subtype of `Element`) for the class `java.io.Externalizable`. I presume at some point it was required for handling serializable/externalizable classes, by checking if it was a supertype of a class being documented.